### DISCUSSION TITLE
Running commands in steam's container from sunshine's?

### DISCUSSION THREAD

**Discussion by omigeot** [games-on-whales/gow#84] [Q&A]
I'm not sure how to do that, now if it's possible (or even desirable), but I'm trying to have several entries in apps.json who launch different "commands" in steam's container. For instance one to open "steam://open/bigpicture", one to open several different games, etc.

So far, IIRC, it just running /usr/games/steam with such URLs won't work in apps.json, since steam runs in a different container. What would be the prefered way of looking into this? Allowing commands to be run between containers (ssh?) ?

**Comment by ABeltramo** [comment]
See this [previous discussion](https://github.com/games-on-whales/gow/discussions/55) which should answer your questions. Feel free to ask if you need any more information!
